oeps/oep-0046-arch-ddd-domains.rst

```diff
@@ -0,0 +1,486 @@
+####################################
+OEP-46: Domain-Driven-Design Domains
+####################################
+
+.. list-table::
+  :widths: 25 75
+
+  - - OEP
+    - :doc:`OEP-46 </oeps/oep-0046-arch-ddd-domains.rst>`
+  - - Title
+    - Domain-Driven-Design (DDD) Domains
+  - - Last Modified
+    - 2020-04-28
+  - - Authors
+    - Cale Pennington <cale@edx.org>
+  - - Arbiter
+    - <Arbiter's real name and email address>
+  - - Status
+    - Draft
+  - - Type
+    - Architecture
+  - - Created
+    - 2020-04-23
+  - - Review Period
+    - 2020-04-23 - 2020-05-30
```

Comment:
It is confusing whether this is in review or not.  If not, do you want to clear this field for now?  Thank you.